### ISSUE TITLE
Fórmula nuevo estado

### ISSUE THREAD

**Issue by cristinalopezamado** [IIC2613-Inteligencia-Artificial-2022-2/Syllabus#111]
En el aviso de canvas con los cambios en la tarea, se indica que la fórmula para el nuevo qvalue es:

![image](https://user-images.githubusercontent.com/57901545/205702647-de28d4ed-10fd-440f-afd5-7f387346426d.png)

No entiendo por qué se calcula argmax Q[x_i]. ¿No habría que tomar el Q[x_i] correspondiente a la acción que se eligió?

**Comment by dfloreaa** [comment]
Claro, esta fórmula considera que se jugó la acción con mayor q-value (explotamos, no exploramos), en caso contrario, habría que actualizar en base a la acción elegida

**Comment by dfloreaa** [comment]
La fórmula general sería algo así:
$new\ qvalue = (1 - \alpha) \cdot Q[accion\ elegida] + \alpha \cdot (reward + \gamma \cdot argmax Q[x_{i+1}])$

Con $Q[accion\ elegida]$ la calidad de la acción ejecutada anteriormente
